src/Microsoft.AspNetCore.Server.Kestrel/Filter/LoggingFilterKestrelOptionsExtensions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Filter
```

Comment:
Now that I think about it, it doesn't matter if this is in the same namespace as the options type because it's used inside a lamda.